### PR TITLE
ahkohd.switch version 1.0.23.95

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24372)

### CODE REVIEW

**Review by ImJoakim — CHANGES_REQUESTED** [review]
image

File: manifests/a/ahkohd/switch/1.0.23.95/ahkohd.switch.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ahkohd.switch
+PackageVersion: 1.0.23.95
```
Comment (ImJoakim) [inline manifests/a/ahkohd/switch/1.0.23.95/ahkohd.switch.installer.yaml:5]: image

```suggestion
PackageVersion: 1.0.23
```

File: manifests/a/ahkohd/switch/1.0.23.95/ahkohd.switch.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ahkohd.switch
+PackageVersion: 1.0.23.95
```
Comment (ImJoakim) [inline manifests/a/ahkohd/switch/1.0.23.95/ahkohd.switch.locale.en-US.yaml:5]: image

```suggestion
PackageVersion: 1.0.23
```

File: manifests/a/ahkohd/switch/1.0.23.95/ahkohd.switch.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: ahkohd.switch
+PackageVersion: 1.0.23.95
```
Comment (ImJoakim) [inline manifests/a/ahkohd/switch/1.0.23.95/ahkohd.switch.yaml:5]: image

```suggestion
PackageVersion: 1.0.23
```

**Comment by ImJoakim** [comment]
Please see my comments above. You also need to change the folder name for the version.

https://github.com/microsoft/winget-pkgs/pull/24372#discussion_r685942495
https://github.com/microsoft/winget-pkgs/pull/24372#discussion_r685942534
https://github.com/microsoft/winget-pkgs/pull/24372#discussion_r685942586

`manifests/a/ahkohd/switch/1.0.23.95/` -> `manifests/a/ahkohd/switch/1.0.23/`

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**